docs/tutorials/services/plan-motion-with-arm-gripper.md

```diff
@@ -20,19 +20,26 @@ cost: 8400
 no_list: true
 ---
 
-{{< alert title="Caution" color="caution" >}}
-Be careful when instructing robot arms to move.
-Before running any code, ensure your robotic arm has enough space and that there are no obstacles.
-Also pay attention to your surroundings, double-check your code for correctness, and make sure anyone nearby is aware and alert before issuing commands to your robot.
-{{< /alert >}}
+With Viam you can move individual components, like [an arm](../accessing-and-moving-robot-arm/) by issuing command like `MoveToPosition()` or `MoveToJointPosition`.
+The [motion service](/mobility/motion/) enables you to do much more sophisticated movement involving one or many components of your robot.
+The service abstracts the lower level commands away so that you can call the `Move()` command with the desired destination position in a frame, alongside any obstacles and geometries attached to your machine, and the service will move your machine to the desired location for you.
```

Comment:
I think this can focus more on the general concept without needing to talk about frames or geometries yet.
```suggestion
The service abstracts the lower-level commands away so that instead of passing in a series of joint positions, you can call the `Move()` command with the desired destination and any obstacles, and the service will move your machine to the desired location for you.
```